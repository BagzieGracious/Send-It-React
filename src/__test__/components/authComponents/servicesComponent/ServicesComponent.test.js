import React from 'react';
import { shallow } from 'enzyme';
import ServicesComponent from '../../../../components/authComponents/servicesComponent';

describe('ServicesComponent test', () => {
	it('should render without crashing', () => {
		const wrapper = shallow(<ServicesComponent />);
		expect(wrapper).toMatchSnapshot();
	});
});
