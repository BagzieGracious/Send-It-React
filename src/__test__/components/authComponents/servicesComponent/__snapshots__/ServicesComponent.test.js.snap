// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ServicesComponent test should render without crashing 1`] = `ShallowWrapper {}`;
